src/NServiceBus.NLog/NLogFactory.cs

```diff
@@ -6,9 +6,12 @@
     /// <summary>
     /// Configure NServiceBus logging messages to use NLog.  Use by calling <see cref="LogManager.Use{T}"/> the T is <see cref="NLogFactory"/>.
     /// </summary>
+    [ObsoleteEx(
+        Message = "Support for external logging providers is no longer provided by NServiceBus providers for each logging framework. Instead, the NServiceBus.Extensions.Logging library provides the ability to use any logging provider that conforms to the Microsoft.Extensions.Logging abstraction.",
```

Comment:
I don't feel like this message provides a clear action for the user. How about something like:

```suggestion
        Message = "NServiceBus is now providing support for logging libraries through the Microsoft.Extensions.Logging abstraction. Remove the NServiceBus.NLog package. Install the NServiceBus.Extensions.Logging and NLog.Extensions.Logging packages instead.",
```